frontend/javascripts/oxalis/view/right-border-tabs/trees_tab/skeleton_tab_view.tsx

```diff
@@ -1009,7 +1009,7 @@ const mapDispatchToProps = (dispatch: Dispatch<any>) => ({
   },
 
   onDeleteTree() {
-    dispatch(deleteTreeAsUserAction());
+    dispatch(handleDeleteTreeByUser());
```

Comment:
```suggestion
    handleDeleteTreeByUser();
```

because null is dispatched otherwise (doesn't crash probably but doesn't make sense either)